conf/ynh-hotspot

```diff
@@ -327,7 +327,7 @@ if [ "$1" != restart ]; then
   done
 
   old_internet_device=$(ynh_setting_get hotspot internet_device)
-  new_internet_device=$(ip route | awk '/default via/ { print $NF; }')
+  new_internet_device=$(ip --json route | jq -r '.[] | select(.dst == "default") | .dev')
```

Comment:
Ogod we found some of these `print $NF` in vpnclient's code recently, didnt thought there was in hotspot too >_> ...

Note that on my laptop, this output two interfaces because I have two routes with dst = default

```
 > ip route
default via 192.168.0.1 dev wlp3s0 proto dhcp metric 600 
default dev enp0s25 scope link metric 1002 linkdown 
[...]
```

I don't know what the second line means, but I'm guessing that we are though looking at the "default via" route ?

The json parsing is pretty nice (didn't know about `ip --json` !) but what about simply using `ip route | awk '/default via/ { print $5; }'` ?

Reply:
Mmh ok, well, as you wish! `ip route | awk '/default via/ { print $5; }'` works fine indeed but it will maybe break one day.

For cases where we do have several default routes, I guess we'll always still have only one gateway (`via 192.168.0.1` in your example) so I would propose the robust way: 
```
new_internet_device=$(ip --json route | jq -r '.[] | select((.dst == "default") and (.gateway != null)) | .dev')
```

Reply:
Whith this version what if there are routes like 0.0.0.0/1 ?
The ideal should be the interface listed with `ip route get 1.2.3.4` but json doesn't work with this command...

Reply:
Ok you're both right. I changed the line. If interface is no more at column 5 one day, well, we'll see :+1: